generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -256,11 +256,15 @@ public User getUserWithAuthorities(<%= pkType %> id) {
     @Transactional(readOnly = true)
     <%_ } _%>
     public User getUserWithAuthorities() {
-        User user = userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin()).get();
-        <%_ if (databaseType == 'sql') { _%>
-        user.getAuthorities().size(); // eagerly load the association
-        <%_ } _%>
-        return user;
+        Optional<User> optionalUser = userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin());
+        User user = null;
+        if (optionalUser.isPresent()) {
+          <%_ if (databaseType == 'sql') { _%>
+            user.getAuthorities().size(); // eagerly load the association
```

Comment:
I think user will be `null` here -> maybe do `user = optionalUser.get()` before?